### ISSUE TITLE
Pathname#/ alias is missing

### ISSUE THREAD

**Issue by amyspark** [oracle/truffleruby#2178]
The `/ ` method (an alias to `+`) is missing from the `Pathname` class. git blaming shows that it was added in MRI [in 2.2.0](https://github.com/ruby/ruby/commit/391fc2eeecfc2f0d0f851b0c27ba02e041fe2cd9).

MWE (from `irb`):

```
irb(main):001:0> require 'pathname'
=> true
irb(main):002:0> x = Pathname("/home")
=> #<Pathname:/home>
irb(main):003:0> x / "amalia"
Traceback (most recent call last):
        9: from /home/amalia/.rbenv/versions/truffleruby-20.3.0/bin/irb:37:in `<main>'
        8: from <internal:core> core/kernel.rb:401:in `load'
        7: from <internal:core> core/kernel.rb:401:in `load'
        6: from /home/amalia/.rbenv/versions/truffleruby-20.3.0/lib/gems/gems/irb-1.0.0/exe/irb:11:in `<top (required)>'
        5: from <internal:core> core/throw_catch.rb:36:in `catch'
        4: from <internal:core> core/throw_catch.rb:36:in `catch'
        3: from <internal:core> core/kernel.rb:438:in `loop'
        2: from <internal:core> core/kernel.rb:325:in `eval'
        1: from (irb):3
NoMethodError (undefined method `/' for #<Pathname:/home>)
irb(main):004:0> x + "amalia"
=> #<Pathname:/home/amalia>
```

Using truffleruby 20.3.0 from rbenv.

**Comment by eregon** [comment]
Indeed, thanks for the report.

**Comment by eregon** [comment]
@amyspark In case you would be interested to contribute a fix, this issue as well as #2179 should actually be very easy to fix because they only require to change Ruby code, and to add a test/spec written in Ruby.
See https://github.com/oracle/truffleruby/blob/master/CONTRIBUTING.md for details about contributing, we're happy to help on the Slack channel.
Of course it's perfectly fine if you prefer to report issues and let someone else fix them, just let me know.

**Comment by bjfish** [comment]
This is now fixed at: https://github.com/oracle/truffleruby/commit/2bc84bd6f97860e0daf5453b74acba4edc9365de
